travis/upload_build_results_to_gcs.py

```diff
@@ -125,22 +132,48 @@ def _DecryptGCPServiceFileTo(service_file_path: str):
             e.__class__.__name__, redacted_message))
 
 
-def _UploadBuildResults(gcs_bucket: storage.Bucket, gcs_build_results_dir: str):
-  """Uploads all build results to Google Cloud Storage."""
-  logging.info("Will upload build results to gs://%s/%s.",
-               os.environ[_GCS_BUCKET], gcs_build_results_dir)
-
-  for build_result in os.listdir(flags.FLAGS.build_results_dir):
-    path = os.path.join(flags.FLAGS.build_results_dir, build_result)
+def _UploadDirectory(local_dir:str, gcs_bucket: storage.Bucket, gcs_dir: str):
+  """Upload the contents of a local directory to a GCS Bucket."""
+  for file_name in os.listdir(local_dir):
+    path = os.path.join(local_dir, file_name)
     if not os.path.isfile(path):
       logging.info("Skipping %s as it's not a file.", path)
       continue
     logging.info("Uploading: %s", path)
-    gcs_blob = gcs_bucket.blob("{}/{}".format(gcs_build_results_dir,
-                                              build_result))
+    gcs_blob = gcs_bucket.blob("{}/{}".format(gcs_dir, file_name))
```

Comment:
Why not f-strings?

Reply:
The `str.format(...)` was inherited from the previous version of the code. I replaced this instance with an f string and I will replace the other instances in this file in a separate PR. Thank you!